src/jdk.crypto.ec/share/classes/sun/security/ec/SunEC.java

```diff
@@ -259,14 +216,7 @@ void putEntries() {
         StringBuilder names = new StringBuilder();
 
         Collection<? extends NamedCurve> supportedCurves;
-        if (SunEC.isNativeDisabled()) {
-            supportedCurves = Collections.unmodifiableList(List.of(
-                    CurveDB.lookup("secp256r1"),
-                    CurveDB.lookup("secp384r1"),
-                    CurveDB.lookup("secp521r1")));
-        } else {
-            supportedCurves = CurveDB.getSupportedCurves();
-        }
+        supportedCurves = CurveDB.getSupportedCurves();
```

Comment:
Shouldn't the supportedCurves be the hardcoded 3 curves?

Reply:
Ah yes.  Thanks for seeing that.